integrations/dbt/bigquery/setup.cfg

```diff
@@ -3,13 +3,13 @@ current_version = 0.16.0
 commit = False
 tag = False
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?P<rc>.*)
-serialize = 
+serialize =
 	{major}.{minor}.{patch}{rc}
 	{major}.{minor}.{patch}
 
 [bumpversion:file:setup.py]
-search = version="{current_version}"
-replace = version="{new_version}"
+search = __version__ = "{current_version}"
+replace = __version__ = "{new_version}"
```

Comment:
Oh, nice! I think we can clean this up even more and update the other `py` modules to follow the convention. For now, this looks great 👍